### ISSUE TITLE
Linear Mixed Models - Satterhtwaite Approximate Degrees of Freedom in SAS/R

### ISSUE THREAD

**Issue by henriettebjrn** [PSIAIMS/CAMIS#297]
Under certain circumstances, the same simulated datasets give differing conclusions in hypothesis tests between lmer in R and proc mixed in SAS due to differences in model fits and Satterthwaite degrees of freedom method. Would documenting this be of value?

**Comment by DrLynTaylor** [comment]
Yes we definitely would love this to be written up and put onto the repository.  Which category do you see this best fitting into, or would you like an additional method category to be added?     I assume it could go under MMRM?   We currently have a lot of work to do on our MMRM section (the R and SAS examples, are a little disjointed to the R vs SAS page).   Is this something you'd be interested in helping with updating and then adding in your Satterhwaite DF issue to it?

**Comment by henriettebjrn** [comment]
At this stage, we prefer not to contribute to updating the MMRM section, but rather to add a section on the Satterthwaite story. We suggest to add a line in the section “Repeated Measures”, perhaps just under the “Linear Mixed Model (MMRM)”. It could be called “Linear Mixed Model (degrees of freedom)” and we would then make a document called “R versus SAS”.

Maybe at a later stage, we could contribute further to the page.

**Comment by DrLynTaylor** [comment]
Hi @henriettebjrn, As we approach the end of year we are checking in on progress to see if anyone needs any help?  Just wondered how this issue was progressing.  Totally fine if work just got in the way and you need more time. Just let us know though if you are having any issues that we can help with.

**Comment by henriettebjrn** [comment]
Hi! Work has been quite busy recently, so unfortunately, progress on this has been delayed a bit. We’re planning to pick it back up around late January.

**Comment by DrLynTaylor** [comment]
Totally understand, that's no problem at all. Thanks for the update.

**Comment by DrLynTaylor** [comment]
@henriettebjrn Are you still working on this issue?   We believe mmrm fixes the lmer issue, but it would still be really good to explain this problem for lmer and that mmrm fixes the issue.

**Comment by henriettebjrn** [comment]
Sorry for the late response. We have made a pull request for mmrm df. Hopefully, the procedure has been done correctly.
